test/integration/standard_filter_test.rb

```diff
@@ -358,6 +358,14 @@ def test_append
     assert_template_result('bcd',"{{ a | append: b}}",assigns)
   end
 
+  def test_concat
+    assert_equal([1,2,3,4 ], @filters.concat([1,2], [3,4] ))
+    assert_equal([1,2,'a' ], @filters.concat([1,2], ['a'] ))
+    assert_equal([1,2, 10 ], @filters.concat([1,2], [ 10] ))
+    assert_equal(['a', 10 ], @filters.concat('a',  10  ))
```

Comment:
Hmm. Still not sure if I feel like this is too much magic or not. I can already see people "misusing" this feature to "create" Array objects. Not sure if that's something that we want. Imho ```concat``` should only work if both inputs are an array, since that is what Ruby does. If you want to append an element to an existing array, you should use the ```append``` filter.